model/model.cc

```diff
@@ -25,6 +25,7 @@
 
```

Comment:
Better to leave a commit message that wouldn't upset whoever originally wrote this code. How about something like:

#undef min/max macros to prevent conflicts